test/infrastructure/docker/templates/cluster-template-development.yaml

```
@@ -34,4 +34,8 @@ spec:
       - class: default-worker
         name: md-0
         replicas: ${WORKER_MACHINE_COUNT}
+      machinePools:
+      - class: default-mp-worker
```

```suggestion
      - class: default-worker
```
Let's use the same. The class for MD/MP should be separate anyway. So by using the same we can confirm that we don't have bugs there